R/cochranecommon.R

```diff
@@ -410,7 +410,7 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
 
   return()
 }
-.cochraneCreateDatabaseKeywords <- function(jaspResults, options, maxKeywords = 500) {
+.cochraneCreateDatabaseKeywords <- function(jaspResults, options, maxKeywords = Inf) {
```

Comment:
Does the speed of this function also contribute to the slowness of the keywords? If yes, there are some performance improvements possible. If not, it's probably unnecessary to optimize this.

Reply:
I don't think so, I just executed it in the console and it's nearly instantaneous.